contracts/staking/PoSValidatorManager.sol

```diff
@@ -57,25 +81,45 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         onlyInitializing
     {
         __ValidatorManager_init(settings.baseSettings);
-        __POS_Validator_Manager_init_unchained(
-            settings.minimumStakeAmount,
-            settings.maximumStakeAmount,
-            settings.minimumStakeDuration,
-            settings.rewardCalculator
-        );
+        __POS_Validator_Manager_init_unchained({
+            minimumStakeAmount: settings.minimumStakeAmount,
+            maximumStakeAmount: settings.maximumStakeAmount,
+            minimumStakeDuration: settings.minimumStakeDuration,
+            minimumDelegationFeeBips: settings.minimumDelegationFeeBips,
+            maximumStakeMultiplier: settings.maximumStakeMultiplier,
+            rewardCalculator: settings.rewardCalculator
+        });
     }
 
     // solhint-disable-next-line func-name-mixedcase
     function __POS_Validator_Manager_init_unchained(
         uint256 minimumStakeAmount,
         uint256 maximumStakeAmount,
         uint64 minimumStakeDuration,
+        uint256 minimumDelegationFeeBips,
+        uint8 maximumStakeMultiplier,
         IRewardCalculator rewardCalculator
     ) internal onlyInitializing {
         PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+        require(minimumDelegationFeeBips > 0, "PoSValidatorManager: zero delegation fee");
+        require(
+            minimumDelegationFeeBips <= MAXIMUM_DELEGATION_FEE_BIPS,
+            "PoSValidatorManager: invalid delegation fee"
+        );
+        require(
+            minimumStakeAmount <= maximumStakeAmount,
+            "PoSValidatorManager: invalid stake amount range"
+        );
+        require(
+            maximumStakeMultiplier <= MAXIMUM_STAKE_MULTIPLIER_LIMIT && maximumStakeMultiplier > 0,
+            "PoSValidatorManager: invalid maximum stake multiplier"
+        );
```

Comment:
Can we split this into two separate require statements? I find it much easier to debug if I know which condition is failing.